net/mdns-repeater/src/opnsense/mvc/app/models/OPNsense/MDNSRepeater/MDNSRepeater.xml

```diff
@@ -16,5 +16,9 @@
       <Required>Y</Required>
       <Multiple>Y</Multiple>
     </interfaces>
+    <blocklist type="TextField">
```

Comment:
You could use a NetworkField

https://docs.opnsense.org/development/frontend/models_fieldtypes.html#networkfield

Also is it bloklist or blocklist?

Here examples:
https://github.com/opnsense/plugins/blob/edae632a23c1c35c518539989373187bdd87c520/www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml#L400-L406

https://github.com/opnsense/plugins/blob/edae632a23c1c35c518539989373187bdd87c520/www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/dialogAccessList.xml#L8-L15

https://github.com/opnsense/plugins/blob/edae632a23c1c35c518539989373187bdd87c520/www/caddy/src/opnsense/service/templates/OPNsense/Caddy/Caddyfile#L86